CIP-0030/README.md

```diff
@@ -81,35 +81,74 @@ transaction_unspent_output = [
 
 then we define
 
-```
+```cddl
 type TransactionUnspentOutput = cbor<transaction_unspent_output>
 ```
 
 This allows us to use the output for constructing new transactions using it as an output as the `transaction_output` in the Shelley Multi-asset CDDL does not contain enough information on its own to spend it.
 
 #### Paginate
 
-```
+```ts
 type Paginate = {|
   page: number,
   limit: number,
 |};
 ```
+
 Used to specify optional pagination for some API calls. Limits results to {limit} each page, and uses a 0-indexing {page} to refer to which of those pages of {limit} items each. dApps should be aware that if a wallet is modified between paginated calls that this will change the pagination, e.g. some results skipped or showing up multiple times but otherwise the wallet must respect the pagination order.
 
 #### Extension
 
-An extension is an object with a single field `"cip"` that describe a CIP number extending the API (as a plain integer, without padding). For example:
+An extension is an object with a single field `"cip`"` that describes a CIP number extending the API (as a plain integer, without padding). For example:
 
-```
+```json
 { "cip": 30 }
 ```
 
+##### Draft or Experimental Extensions
+
+Extensions that are draft, in development, or prototyped should not use official "cip" numbering.
+
+They should instead be referred to as "x-cip" and the number can be arbitrary and should not be expected to be unique.
```

Comment:
I think the term "x-cip" is somewhat ambiguous, I see no reason we could use something like "experimental" or "draft-cip".

Reply:
yes, when I saw the prefix `x-` that is exactly what I thought: based on the MIME convention.  So it seems intuitive & well founded to me.

Reply:
Ahh, I didn't know about that :)

Reply:
For a quick view, view full headers in any email message :nerd_face: